rspec-mode.el

```diff
@@ -466,6 +480,28 @@ buffers concurrently"
   (rspec-run-single-file (rspec-spec-file-for (buffer-file-name))
                          (rspec-core-options)))
 
+(defvar rspec-tags-history nil
+  "History of tags used in rspec-verify-tags-* functions.")
+
+(defun rspec-add-tags-to-history (tags)
+  "Add TAGS to rspec-tags-history and optionally save to savehist."
+  (dolist (tag tags)
+    (add-to-history 'rspec-tags-history tag))
+  (when (and (featurep 'savehist) savehist-mode)
+    (unless (member 'rspec-tags-history savehist-additional-variables)
+      (add-to-list 'savehist-additional-variables 'rspec-tags-history))))
+
+(defun rspec-verify-tags ()
+  "Run the specified spec, or the spec file for the current buffer, filtered by one or multiple tags."
+  (interactive)
+  (rspec--autosave-buffer-maybe)
+  (let* ((tags-string (completing-read-multiple "Select tags (separated by comma): " rspec-tags-history))
```

Comment:
I think it's better to use the history variable as the HIST argument in `completing-read-multiple`. It's the argument number 6. Doing that would automatically add the selected input to history as well (so `rspec-add-tags-to-history` won't be needed, I think).

I guess you could pass `rspec-tags-history` as `TABLE` and `'rspec-tags-history` (quoted) as the history variable, so both saving to history works, and completion works as well.